templates/deployments.yaml

```
@@ -976,6 +981,11 @@ spec:
       annotations:
         checksum/config: {{ include (print $.Template.BasePath "/configmaps_st2-conf.yaml") . | sha256sum }}
         checksum/ssh: {{ include (print $.Template.BasePath "/secrets_ssh.yaml") . | sha256sum }}
+        {{- if .Values.st2actionrunner.annotations }}
+        {{- range $key, $value := .Values.st2actionrunner.annotations }}
+        {{ $key }}: {{ $value | quote }}
+        {{- end }}
```

Is there any advantage of iterating comparing to more simple construct?

Can we use just use
```suggestion
{{ toYaml .Values.st2actionrunner.annotations | indent 8 }}
```
?

---

Definitely we should use what you propose, I was just not sure it works (I looked for a similar construct and I only encountered the iteration approach). Thank you!